legend-shared-pac4j/src/main/java/org/finos/legend/server/pac4j/hazelcaststore/HazelcastSessionDetails.java

```diff
@@ -0,0 +1,48 @@
+// Copyright 2023 Goldman Sachs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.finos.legend.server.pac4j.hazelcaststore;
+
+import java.io.Serializable;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+
+public class HazelcastSessionDetails implements Serializable
+{
+
+    private final Date createdDate;
```

Comment:
Do we need this? If not, let's leave it out. If so, let's replace java.util.Date with java.time.Instant.

Reply:
I've removed it